libbeat/cmd/instance/metrics.go

```diff
@@ -30,3 +36,12 @@ func (memstatsVar) Visit(m monitoring.Mode, V monitoring.Visitor) {
 		monitoring.ReportInt(V, "gc_next", int64(stats.NextGC))
 	}
 }
+
+func (fixedstatsVar) Visit(m monitoring.Mode, V monitoring.Visitor) {
+	uptime := int64(time.Now().Sub(startTime).Seconds())
+
+	V.OnRegistryStart()
+	defer V.OnRegistryFinished()
+
+	monitoring.ReportInt(V, "uptime", uptime)
```

Comment:
Can we send `uptime.ms` (number of milliseconds since unix epoch)? This is how we send uptime in the [system-uptime metricset](https://github.com/elastic/beats/blob/bbc28b08c891d7aab1c6a13c64570a8536a40cdd/metricbeat/module/system/uptime/_meta/fields.yml#L6-L11) and in the new windows-service metricset.

Reply:
@tsullivan what's the expected format for uptime?